packages/owl-bot/src/copy-code.ts

```diff
@@ -272,7 +272,7 @@ export function copyDirs(
     deadPath = path.join(destDir, deadPath);
     if (stat(deadPath)) {
       logger.info(`rm -r ${deadPath}`);
-      fs.rmSync(deadPath, {recursive: true});
+      fs.rmdirSync(deadPath, {recursive: true});
```

Comment:
`rmSync` lands in `14.x` I believe, this was breaking for me locally. `rmdirSync` with recursive is equivalent (and not deprecated until `16.x`).